app/__init__.py

```
@@ -0,0 +1,10 @@
+from fastapi import FastAPI
+from .routes import router
+
+app = FastAPI(
+    title="OpenAPI Comparison Service",
+    description="Service for comparing OpenAPI specifications between different commits",
+    version="1.0.0"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)